go/apps/http_api_nostream/tests/test_vumi_app.py

```diff
@@ -272,6 +286,28 @@ def assert_metrics_published(self, metrics, prefix=None):
             [("%s.%s" % (prefix, name), value, agg)
              for name, value, agg in metrics])
 
+    def assert_retry(self, retry, url, method='POST', owner='test-0-user',
+                     intervals=(300, 1800, 3600)):
+        self.assertEqual(retry.method, "POST")
```

Comment:
Perhaps I need to make it clearer somehow, but there are two different
requests that are relevant here -- the request to be retried and request to
the retry server. The `method` parameter refers to the method of the
request to be retried. The method of the call to the retry server is always
'POST.